src/olympia/lib/settings_base.py

```diff
@@ -1286,10 +1286,6 @@ def get_db_config(environ_var, atomic_requests=True):
     ANALYTICS_HOST,
     PROD_CDN_HOST,
 )
-CSP_FORM_ACTION = (
-    "'self'",
-    'https://developer.mozilla.org',
```

Comment:
actually, can we change this to just drop the `https:// ....` line, and keep the rest of the tuple as now.